feature/reward/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,34 @@
+<resources>
+    <!-- Reward -->
+    <string name="reward_title">Reward</string>
+    <string name="format_consume_point">%dP</string>
+    <string name="reward_empty_text">Please add new reward</string>
+    <string name="confirm_title">Confirm</string>
+    <string name="reward_delete_message">%s was Deleted</string>
+    <string name="delete_confirm_message">Do you delete %s?</string>
+    <string name="error_acquire_reward">You need more points</string>
+    <string name="ticket_with_unit">%d Tickets</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/4d0ed9730273304795d5000d7f91c35c624f32af/feature/reward/src/main/res/values/strings.xml#L10"></span>Formatting %d followed by words ("Tickets"): This should probably be a plural rather than a string</td>
    </tr>
  </tbody>
</table>